extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -270,6 +271,13 @@ public void onConnectionLost(HueBridge bridge) {
     public void onConnectionResumed(HueBridge bridge) {
         logger.debug("Bridge connection resumed. Updating thing status to ONLINE.");
         updateStatus(ThingStatus.ONLINE);
+        // now also re-initialize all light handlers
+        for (Thing thing : getThing().getThings()) {
+            ThingHandler handler = thing.getHandler();
+            if (handler != null) {
+                handler.initialize();
```

Comment:
The initialize method (that is called by the framework) needs to set the thing to offline or online.
So, if there is another logic to use, tthe initialize logic should set the things to offline.
If there is a need, that the polling loop is run at least once, then a new function (as @kaikreuzer already pointed out) could be called later, that sets the state to online.
This function could be called by the initialize method if the polling has already be run, too.